docs/code-reviews.md

```diff
@@ -1,4 +1,67 @@
-!!! note "These documents are a work-in-progress"
-    This section needs to be written! A "code review" is an important quality assurance practice. Every time you make a Pull Request in Github, you should ask a collaborator to do a review. [This video](https://github.com/features/code-review/) is a good starting point.
+## What is a code review?
 
----8<-- 'includes/glossary.md'
+A code review is a critical quality assurance practice. It involves systematically
+checking a fellow programmer's (or researcher's) code for errors or possible 
+improvements. 
+
+It is an everyday activity in the software development world and improves
+both quality and speed in programming. 
+
+## How to ask for a review
+
+Github has a feature built into the workflow that allows easy code review with collaborators, and we would recommend 
+that you use this tool. 
+
+![image](https://user-images.githubusercontent.com/25401512/109499428-cdb2f680-7a8c-11eb-9d77-4e53f7f1c57c.png)
+
+In this screenshot, the user is making a new Pull Request. In the top right-hand corner (marked by the 
+red box), there is a tagging function under Reviewers where you can search or select collaborators to do a code review. 
+Once you have tagged them and made the Pull Request, the person will receive a notification to their 
+email, and it will also come up in the Pull Request section of their Github main screen. 
+
+![image](https://user-images.githubusercontent.com/25401512/109499512-e91e0180-7a8c-11eb-8f7a-5024f7753366.png)
+
+It is strongly encouraged to add some information to 
```

Comment:
Link to a good PR? For example I've noticed Anna has done some good ones last week?

Reply:
Or a screenshot of a good PR even better (which is also a link to the original). Pictures are good